src/modules/esl-event-listener/core/targets/decorated.target.ts

```diff
@@ -51,7 +51,7 @@ export class ESLEventTargetDecorator<Args extends any[]> extends SyntheticEventT
   public override addEventListener(event: any, callback: EventListener = event): void {
     super.addEventListener(event, callback);
 
-    if (!this.hasEventListener(event, 1)) {
+    if (this.getEventListeners(event).length < 2) {
```

Comment:
```suggestion
    if (this.getEventListeners(event).length > 1) return;
```
Let's implement the check consistent across the places